DOCS-162 - Add new UI hidden instructions to Metrics (Kanso) (#4065)
SumoLogic/sumologic-documentation: docs/metrics/manage-metric-volume/disabled-metrics-sources.md

* `label.com.docker.swarm.task.id`
* `label.com.docker.swarm.task.name`

> For other dimensions that exceed the cardinality limit Sumo generates a metric transformation rule to exclude the dimension. The rule appears in the UI in **Metrics Transformation Rules**.

Reviewer: ```suggestion
For other dimensions that exceed the cardinality limit, Sumo generates a metric transformation rule to exclude the dimension. The rule appears in the UI in **Metrics Transformation Rules**.
```

> <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso-->  To access **Metrics Transformation Rules**, in the main Sumo Logic menu select **Manage Data > Metrics > Metrics Transformation Rules**. 

Reviewer: ```suggestion
<!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso-->  To access **Metrics Transformation Rules**, go to the main Sumo Logic menu and select **Manage Data > Metrics > Metrics Transformation Rules**. 
```

<!--Kanso 

> [**New UI**](/docs/get-started/sumo-logic-ui-new/). To access **Metrics Transformation Rules**, in the top menu select **Configuration** and then under **Metrics** select **Metrics Transformation Rules**. You can also click the **Go To...** menu at the top of the screen and select **Metrics Transformation Rules**. 

Reviewer: ```suggestion
[**New UI**](/docs/get-started/sumo-logic-ui-new/). To access **Metrics Transformation Rules**, go to the top menu select **Configuration**, and then under **Metrics**, select **Metrics Transformation Rules**. Alternatively, you can click the **Go To...** menu at the top of the screen and select **Metrics Transformation Rules**. 
```
